lib/ssf/client.rb

```diff
@@ -38,7 +39,7 @@ def send_to_socket(span)
       end
     end
 
-    def start_span(name: '', tags: {}, parent: nil, indicator: false)
+    def start_span(name: '', tags: {}, parent: nil, indicator: false, override_service: nil)
```

Comment:
I'm not really a fan of `override_*` as a name - can we just name this `service:`?

Reply:
Using `service` means it conflicts with the `attr_reader :service`, which is a bit awkward.